README.md

```diff
@@ -9,6 +9,14 @@ Powertools for AWS Lambda (Java) is a developer toolkit to implement Serverless
 
 **[📜Documentation](https://docs.powertools.aws.dev/lambda-java/)** | **[Feature request](https://github.com/aws-powertools/powertools-lambda-java/issues/new?assignees=&labels=feature-request%2C+triage&template=feature_request.md&title=)** | **[🐛Bug Report](https://github.com/aws-powertools/powertools-lambda-java/issues/new?assignees=&labels=bug%2C+triage&template=bug_report.md&title=)** | **[Detailed blog post](https://aws.amazon.com/blogs/opensource/simplifying-serverless-best-practices-with-aws-lambda-powertools-java/)**
 
+### Java Compatibility
+Powertools for AWS Lambda (Java) supports all Java version from 8 up to 21 as well as the 
+[corresponding Lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). 
+
+[AspectJ does not yet support Java 21](https://github.com/eclipse-aspectj/aspectj/issues/260).
+If you need aspect features such as `@Logging` you should use the JDK 17 compiler and target either the Java 17 or Java 21
```

Comment:
What exactly does this mean? Do we have examples?

Reply:
cleaned up wording a bit. Hopefully AspectJ for 21 will be released soon and we can remove this again!